crs:
  general: {}
  motion_planning:
    home_position:
      joint_names: [shoulder_pan_joint, shoulder_lift_joint, elbow_joint, wrist_1_joint, wrist_2_joint, wrist_3_joint]
      joint_position: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    process_path:
      tool_speed: 0.1
      offset_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] # [px, py, pz, rx, ry, rz] relative to default tool
      retreat_dist: 0.03
      approach_dist: 0.03
      tool_frame: "sander_center_link"
    media_change:
      change_time: 300.0 # seconds
      change_pose: [-0.381, -0.202, 1.5, 0.0, -0.786, 1.57] # [px, py, pz, rx, ry, rz] in world coordinates
    preview:
      time_scaling: 1.0
  scan_acquisition: 
    scan_poses:
      - pose: [0.0, -0.6, 1.7, 0.0, 0.8791, 0.4395, -0.1843]  # [px, py, pz, qx, qy, qz, qw]
      - pose: [0.0, -0.53, 1.7, -0.9690091, -0.2422523, 0.3, -0.0483246]
      - pose: [0.0, 0.0, 1.7, -0.9690091, -0.2422523, 0.3, -0.0483246]
    tool_frame: "camera_link_optical" 
    skip_on_failure: true
  process_execution:
    time_tolerance: 5.0 # seconds
    joint_tolerance: [0.035, 0.035, 0.035, 0.035, 0.035, 0.035] # radians
  part_registration:
    target_frame_id: "world"
    part_file: "null"      # relative to known data directory
    toolpath_file: "null"  # relative to known data directory
  part_rework: {}